### PR TITLE
Updated devtool bash files

### PR DESCRIPTION
**Pull request by Laxzal**
- install_arc.sh: Attempts to determine if Mamba is installed and if so, use it as the pkg command; Checks if RMG-Py/RMG-database exist as folders already and if so, pull from the main branch.
- install_pyrms.sh: Currently not in use but has been updated to use the main branch of RMS
- install_RMS_v032.jl: File has been archived.

### CODE REVIEW

File: devtools/install_arc.sh [hunk 1]
```diff
@@ -1,21 +1,50 @@
...
 export PYTHONPATH=$PYTHONPATH:$(pwd)
 echo "export PYTHONPATH=$PYTHONPATH:$(pwd)" >> ~/.bashrc
-mamba env update -n rmg_env -f environment.yml
+
+
+$COMMAND_PKG env update -n rmg_env -f environment.yml
 conda activate rmg_env
 make
 python -c "import julia; julia.install(); import diffeqpy; diffeqpy.install()"
-julia -e 'using Pkg; Pkg.add(PackageSpec(name="ReactionMechanismSimulator",rev="main")); using ReactionMechanismSimulator;'
```
Comment (alongd) [inline devtools/install_arc.sh:18]: In the [docs ](http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/anacondaDeveloper.html#installation-by-source-using-anaconda-environment-for-unix-based-systems-linux-and-mac-osx)they have `using ReactionMechanismSimulator`. Is `Pkg.build("ReactionMechanismSimulator")` better?

File: devtools/install_arc.sh [hunk 2]
```diff
@@ -24,17 +53,17 @@ cd ARC
...
 else
-    mamba env create -f environment.yml
+    $COMMAND_PKG env create -f environment.yml
 fi;
-mamba activate arc_env
+conda activate arc_env
 #make install-all
-###Temp###
-mamba create -n xtb_env python=3.7 -y
+###Temp### To be used until ARC repo install is properly updayed
```
Comment (alongd) [inline devtools/install_arc.sh:62]: Can you open an issue on ARC if it needs to be updated?
(also, small typo here)

File: devtools/install_arc.sh (hunk 1 above)
Comment (Laxzal) [reply devtools/install_arc.sh:18]: In this instance it is kinda the same thing. in this instance. `using ReactionMechanismSimulator` is essntially Julia's way of importing a package like one would do in python - `import ReactionMechanismSimulator`. It isn't doing anything unless you then use the function in a script at that moment.

Also, after this update, I want to create a sys.so image of the julia packages to improve run time. But that's for the next PR

File: devtools/install_arc.sh (hunk 2 above)
Comment (Laxzal) [reply devtools/install_arc.sh:62]: Opened: https://github.com/ReactionMechanismGenerator/ARC/issues/580

**Review by alongd — APPROVED** [review]
Thanks!

**Pull request merged**